yejee/week7/완주하지 못한 선수.java

```diff
@@ -0,0 +1,25 @@
+import java.util.HashMap;
+import java.util.Map;
+
+class Solution {
+    public String solution(String[] participant, String[] completion) {
+        Map<String, Integer> participantMap = new HashMap<>();
+        int count = 1;
+        
+        for(String name : participant){
+            count = 1;
+            if(participantMap.containsKey(name))
+                count = participantMap.get(name) + 1;
+            participantMap.put(name, count);
+        }
```

Comment:
for (String name : participant) {
  int count = 1;
  if (participantMap.containsKey(name)) {
    count = participantMap.get(name) + 1;
  }
  participantMap.put(name, count);
}